source/patterns/04-pages/research-article/research-article.twig

```diff
@@ -31,6 +31,10 @@
           template: 'organisms-content-header',
           arguments: contentHeader,
         },
+        {
+          template: 'molecules-contextual-data',
+          arguments: contextualData,
```

Comment:
Not currently working as expected, but this could be because the `contextual-data` template does not handle attributes properly yet. Revisit once that's fixed.